docs/sources/enterprise/request-security.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Request security"
+description = "Grafana Enterprise request security"
+keywords = ["grafana", "security", "enterprise"]
+weight = 110
++++
+
+# Request security
+
+> **Note:** Available in Grafana Enterprise v7.4+.
+
+Request security introduces ways to limit requests from the Grafana server, and it targets requests that are generated by users.
+
+Example:
+- data source metric queries
+- alert notifications
+
+This can be used to limit access to internal systems that the server Grafana runs on can access but that users of Grafana should not be able to access. This feature does not affect traffic from the Grafana users browser.
+
+Note: Although request security works with backend plugins, it is possible to create a backend plugin that bypasses this security.
```

Comment:
```suggestion
Note: Although request security works with backend plugins, you can create a backend plugin that bypasses this security.
```

Reply:
+1

Reply:
I'm seriously tempted to add "it is possible" and other common wordy expressions to the style guide.